adafruit_debouncer.py

```diff
@@ -24,7 +24,7 @@
 ====================================================
 
 Debounces an arbitrary predicate function (typically created as a lambda) of 0 arguments.
-Since a very common use is debouncing a digital input pin, the initializer accepts a pin number
+Since a very common use is debouncing a digital input pin, the initializer accepts a DigitalInOut object
```

Comment:
Pylint is failing on:
```
Pylint is failing on the following:
```
************* Module adafruit_debouncer
C: 27, 0: Line too long (104/100) (line-too-long)
W:135, 8: Attribute '_interval' defined outside __init__ (attribute-defined-outside-init)
```
This line is too long and must be distributed onto two lines.